python/GangaDirac/Lib/Files/DiracFile.py

```diff
@@ -766,6 +766,9 @@ def put(self, lfn='', force=False, uploadSE="", replicate=False):
                 self.failureReason += '\n' + failureReason
                 continue
 
+            if not isinstance(stdout, dict):
```

Comment:
Would it be better to insert this `raise` into the `execute` method from `GangaDirac.Lib.Utilities.DiracUtilities`? In that way the sanity check will be done for all calls and not just for this specific one.

Reply:
I don't think so - from the Dirac call you could conceivably expect strings or anything. It is just for this specific command we expect a dict.

Reply:
Actually I think everything should come back as a dict - I'll change that and close #1159